openreview/conference/matching.py

```diff
@@ -407,13 +408,15 @@ def _build_config_invitation(self, scores_specification):
                         'default': scores_specification
                     },
                     'aggregate_score_invitation': {
-                        'value': self._get_edge_invitation_id('Aggregate_Score'),
+                        'value-regex': '.*',
```

Comment:
shouldn't the regex starts al least with the conference id?

Reply:
Agreed, code updated accordingly.